### PR TITLE
Update to latest ntfs-3g stable version

### PR DESCRIPTION
**Pull request by z-e-r-o**
Get latest stable ntfs-3g version.

### CODE REVIEW

File: packages/sysutils/ntfs-3g_ntfsprogs/package.mk [hunk 1]
```diff
@@ -1,4 +1,4 @@
-################################################################################
+7###############################################################################
```
Comment (lrusak) [inline packages/sysutils/ntfs-3g_ntfsprogs/package.mk]: whoops ;)

**Comment by lrusak** [comment]
Thanks!

does the patch not apply or is it just not needed anymore?

**Comment by z-e-r-o** [comment]
In my opinion not needed anymore. What the patch does is now included. (Somewhere around line 350-355)

File: packages/sysutils/ntfs-3g_ntfsprogs/package.mk (hunk 1 above)
Comment (z-e-r-o) [inline packages/sysutils/ntfs-3g_ntfsprogs/package.mk]: Good first pull request :$
Don't know what went wrong.
I better keep on trying :)

Comment (lrusak) [inline packages/sysutils/ntfs-3g_ntfsprogs/package.mk]: It's OK, I can fix it

**Comment by lrusak** [comment]
whoops. I tried to fix it but I messed it up. Sorry. It won't let me reopen the PR I don't know if it will let you. Otherwise just open a new PR please :)

**Pull request closed without merging**